From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Tom Levy <tomlevy93@gmail.com>
Date: Mon, 27 Jul 2020 08:57:18 +1200
Subject: Redirect all of Composer's output to stdout when building Docker
 image from source

Composer writes normal logging output to stderr. GitLab CI displays
stderr in red, which makes Composer's output very distracting.
---
 docker/domserver/Dockerfile.source | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/docker/domserver/Dockerfile.source b/docker/domserver/Dockerfile.source
index 3b0d255c5..317a21008 100644
--- a/docker/domserver/Dockerfile.source
+++ b/docker/domserver/Dockerfile.source
@@ -50,7 +50,7 @@ RUN php -r "copy('https://getcomposer.org/installer', 'composer-setup.php');" \
 # Install PHP dependencies
 # (also done by "make ... dist" below, but we do it here *before* adding the rest of the source to leverage the build cache)
 COPY --chown=domjudge composer.json composer.lock /domjudge-src/domjudge/
-RUN cd /domjudge-src/domjudge && sudo -u domjudge composer install --prefer-dist -o --no-scripts
+RUN cd /domjudge-src/domjudge && sudo -u domjudge composer install --prefer-dist -o --no-scripts 2>&1
 
 # Add DOMjudge source code
 COPY --chown=domjudge . /domjudge-src/domjudge
-- 
2.20.1

